blark/iec.lark

```diff
@@ -29,6 +29,7 @@ _library_element_declaration: data_type_declaration
                             | program_declaration
                             | global_var_declarations
                             | action
+                            | ";"
```

Comment:
I think this is fine. Some stream-of-consciousness thoughts as I read through this:

* Since this isn't a rule it doesn't get picked up in `blark.transform` or elsewhere. Extraneous semicolons between source code elements then won't be a problem.
* I admittedly have struggled a bit with appropriate handling of semicolons throughout the grammar; you'll see I waffled around with them multiple times, going from `";"` to `";"*` to a mix of `*` and `+`...

Reply:
Hah! That sounds a lot like how I'd gone back-n-forth on some of this stuff! Let me know if you ever choose one way over the other! 😄